src/Plugin.php

```
@@ -180,6 +180,13 @@ private function attachEventHandlers(): void
             UrlManager::EVENT_REGISTER_SITE_URL_RULES,
             function (RegisterUrlRulesEvent $event) {
                 if (!$this->settings->takeoverRouting) {
+
+                    foreach ($event->rules as $pattern => &$rule) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$pattern'. (Unused Code Rules)

(UnusedLocalVariable)